README.md

```diff
@@ -10,49 +8,50 @@
 
 ## Introduction
 
-**ferlab/postprocessing** is a bioinformatics pipeline that ...
+**ferlab/postprocessing** is a bioinformatics pipeline that recombines gvcf for family's samples in order to facilitate denovo identification using tags and annotations. 
```

Comment:
Combine, perform joint genotyping, tag low quality variant and annotate vcf file